docs/reference/migration/migrate_5_0/search.asciidoc

```diff
@@ -128,6 +128,7 @@ in favour of `query` and `no_match_query`.
 
 * Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoPolygonQuery. Use parameter validation_method instead.
 
+* Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoDistanceRangeQuery. Use parameter validation_method instead.
```

Comment:
> This page explains ignore_unmapped, not ignore_malformed. I was under the 
> impression that ignore_malformed and coerce were only used in mappings, not in 
> queries?

Darn - had only skimmed the page really. True, this page doesn't mention it.

It is included in the query options docs here though:

* [Geo Distance Query](https://www.elastic.co/guide/en/elasticsearch/reference/master/query-dsl-geo-distance-query.html)
*[Geo Bounding Box Query](https://www.elastic.co/guide/en/elasticsearch/reference/master/query-dsl-geo-bounding-box-query.html) 


Thanks for the information about how to add deprecation warnings - will add.

> Ignore them.

+1 - will do